hs_model_program/static/js/model_program.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Created by tonycastronova on 10/12/15.
+ */
+
+$alert_error = '<div class="alert alert-danger" id="error-alert"> \
+                    <button type="button" class="close" data-dismiss="alert">x</button> \
+                    <strong>Error </strong> \
+                    failed to retrieve metadata for this resource.\
+                </div>'
+
+
+$(document).ready(function () {
+
+    // check to see if the view is in edit mode
+    if(document.getElementById("id-mpmetadata") != null)
+        // load jquery generated elements
+        loadWidgets();
+
+        // initialize datepicker
+        $( "#modelReleaseDate_picker" ).datepicker({
+            // bind a table view to the on close event
+            onSelect: function (event) {
+                onChange_releaseDate(event);
+            }
+        });
+});
+
+/**
+ * Sets the hidden date field base on the datepicker.  The hidden field is what is submitted.
+ */
+function onChange_releaseDate(event){
+    var date = document.getElementById('id_modelReleaseDate');
+    var date_picker = document.getElementById("modelReleaseDate_picker");
+    date.value = date_picker.value;
+
+    // activate the save button
+    $("#resourceSpecificTab").find('.btn-primary').show();
+}
```

Comment:
if this is only used above then it should be part of the call to `datepicker()` inline